src/components/Clock.tsx

```diff
@@ -0,0 +1,55 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  today: Date,
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
```

Comment:
i think, if you'll keep in state already transform state, it fix failed tests
but it's not exactly :)
```suggestion
    today: new Date().toUTCString().slice(-12, -4),
```